lib/profileAgents.js

```diff
@@ -120,8 +112,9 @@ exports.create = async ({
   const capabilityAgent = new CapabilityAgent({handle: 'primary', signer: key});
   // 7. Create keystore to store keys controlled by the profile agent
   const paKeystore = await kms.createKeystore({
+    // this is a public keystore that can be access by any IP
```

Comment:
We need a better name than "public keystore". Just eliminating the term here for now since the rest of the description handles it.

```suggestion
    // this is a keystore that can be accessed by any IP
```